server/server.js

```diff
@@ -16,5 +16,10 @@ app.get('/checkoutInformation/:productId', (req, res) => {
     .catch(err => console.log(`error looking for checkout information ${err}`));
 });
 
+app.get('/clearDatabaseUtility', (req, res) => {
```

Comment:
im not following why you have a route to clear the database? would this imply the client will need to clear the db?

Reply:
this won't be here long time it was more for me to clear my database because i was trying to test my seeding thanks for the flag though